src/templates/form/folding.html

```diff
@@ -0,0 +1,24 @@
+<a class="pf-widget-caption">
+	<p class="pf-widget-headline"></p>
+	<span>&rsaquo;</span>
+</a>
+<a class="pf-widget-caption-left">
+	<p class="pf-widget-headline"></p>
+	<span>&rsaquo;</span>
+</a>
+<div class="pf-widget-body">
+</div>
+<div class="pf-widget-content">
+	<p class="pf-widget-message"></p>
+	<div class="pf-social-login">
+		<p name="fb-login" hidden></p>
+		<p name="google-login" hidden></p>
+	</div>
+	<form>
+		<input name="username" type="text">
+		<input name="title" type="text">
+		<input name="email" type="email">
+		<textarea  name="message" rows="5"></textarea>
+		<button class="pf-widget-btn pf-widget-cancel">Cancel</button><button type="submit" class="pf-widget-btn pf-widget-ok">Send</button>
+	</form>
+</div>
```

Comment:
Probably not a big deal, but as far as I can tell everything else in pathfora is two-space indentation, so it might be nice to have the templates the same.